### PR TITLE
Dmpr core update

### PR DESCRIPTION
**Pull request by reisub-de**
updates the simulator to the new core interface, also starts with refactoring the scenarios

### CODE REVIEW

File: scenarios/large_static.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        if rx_router != tx_router:
+            break
+
+    tx_router.is_transmitter = True
+    rx_router.is_receiver = True
+    rx_ip = rx_router.pick_random_configured_network()
+
+    for sec in range(SIMULATION_TIME):
+        if sec % 5 != 0:
+            #continue
```
Comment (hgn) [inline scenarios/large_static.py]: ?

Comment (reisub-de) [reply scenarios/large_static.py]: sorry, forgot to remove that, I was testing "fast-forward"

File: dmprsim.py [hunk 2]
```diff
@@ -59,7 +60,7 @@ def log(self, msg, sev, time=lambda: datetime.now().isoformat()):
 
 
 class Tracer(core.dmpr.NoOpTracer):
-    def __init__(self, directory, enable: list=None):
+    def __init__(self, directory, enable: list = None):
```
Comment (hgn) [inline dmprsim.py]: das ist doch falsch! Also wenn ich mir standard python code der standardbibliothek anschaue ist dies hier ohne whitespace kodiert.

Comment (reisub-de) [reply dmprsim.py]: https://www.python.org/dev/peps/pep-0008/#other-recommendations

> When combining an argument annotation with a default value, use spaces around the = sign (but only for those arguments that have both an annotation and a default).
> 
> Yes:
> 
> ```python
> def munge(sep: AnyStr = None): ...
> def munge(input: AnyStr, sep: AnyStr = None, limit=1000): ...
> ```
> 
> No:
> 
> ```python
> def munge(input: AnyStr=None): ...
> def munge(input: AnyStr, limit = 1000): ...
> ```

**Pull request merged**